src/views/ReportView.vue

```diff
@@ -18,10 +18,18 @@ import Report from '../components/Report.vue'
 import BackButton from '../components/BackButton.vue'
 import { useAtlasStore } from '@/stores/atlas'
 import { storeToRefs } from 'pinia'
+import { onBeforeMount } from 'vue'
 const atlasStore = useAtlasStore()
 const { validMapPixel } = storeToRefs(atlasStore)
 
-// This will parse/set the lat/lng if it's being supplied via URL.
-// TBD << reconstituting this from URL via rewrite/traps
 const props = defineProps(['lat', 'lng'])
+
+onBeforeMount(() => {
```

Comment:
I'm sorry to be pedantic.  But can this move into the `Report` component?  Then the data fetch + clear are inside the same component, which feels good from a code factoring perspective.